spec/mailers/payment_mailer_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "rails_helper"
+
+RSpec.shared_examples "an email related to a payment" do |policy|
```

Comment:
Shared examples are useful for DRYing up specs where there are many things that behave the same way – using a shared example removes the need to duplicate the same specs over and over. This shared example is only used once below, which makes it hard to justify. A couple of alternatives I can think of: 

a) get rid of the shared example here and inline the examples below in the `describe "#payment_confirmation"` block
b) figure out how to change the existing shared example such that it can be re-used in the specs below, rather than duplicating a near identical one. The shared example doesn't need to live in a spec file. There are some conventions for where to store them in the docs — https://relishapp.com/rspec/rspec-core/docs/example-groups/shared-examples

What do you reckon?

Reply:
I don't think we should make the shared example reusable, because the behaviour that it's testing is going to diverge in the payment mailer shortly, once we introduce multiple claims. I think handling both behaviours in a single set of examples would not be worth the complexity.

Reply:
Which is to say: I'll inline them 😄